src/app/qgisapp.h

```diff
@@ -1008,6 +1008,20 @@ class APP_EXPORT QgisApp : public QMainWindow, private Ui::MainWindow
      */
     void triggerCrashHandler();
 
+    /**
+     * Emits the signal to block updates for attribute tables connected a particular \a layer
+     *
+     * \since QGIS 3.4
+     */
+    void blockAttributeTableUpdates( const QgsVectorLayer *layer );
```

Comment:
Yes, I first implemented it like that. But I was in doubt, because in general we don't like bool switches, do we?
But you solved my doubt, so I'm fine with the bool now. (I've always thought that `blockSignals(bool)` was not very consistent with the rest of Qt API).